dateparser/languages/validation.py

```diff
@@ -0,0 +1,310 @@
+# -*- coding: utf-8 -*-
+import re
+from logging import getLogger
+
+
+class LanguageValidator(object):
```

Comment:
I know that it's not actually re-parsing the file, it's just quite like it in the sense that it also reflects the whole structure.

Thanks for the detailed explanation, I understand that the main purpose of this is to increase confidence in consistence configuration across all the languages.

I still think there's a lot of room for improvement here, but I rather have this PR merged already, since this is such a minor concern. :)